Firestore/Source/Remote/FSTDatastore.mm

```diff
@@ -305,20 +305,19 @@ - (void)invokeRPCWithFactory:(GRPCProtoCall * (^)(void))rpcFactory
   // TODO(mikelehen): We should force a refresh if the previous RPC failed due to an expired token,
   // but I'm not sure how to detect that right now. http://b/32762461
   _credentials->GetToken(
-      /*force_refresh=*/false,
-      [self, rpcFactory, errorHandler](Token result, const int64_t error_code,
-                                       const absl::string_view error_msg) {
-        NSError *error = util::WrapNSError(error_code, error_msg);
+      /*force_refresh=*/false, [self, rpcFactory, errorHandler](util::StatusOr<Token> result) {
+        NSError *error = util::WrapNSError(result.status());
```

Comment:
We've been calling these utilities two different things: "wrap parameter" or "make return type". We have a few cases where we did "wrap return type" but these seem to have been in error (mostly stemming from `WrapNSStringNoCopy` really wanting to convey that it was well and truly wrapping the underlying string.

I don't feel terribly strongly about this, but it seems like we could move toward a standard naming for this. Without expressing too strong a preference either way, how about `MakeNSError`?

Reply:
I agree that `WrapNSError` sounds as if an existing `NSError` were wrapped in some object. Since this is a conversion, I can also suggest `ToNSError` or `AsNSError` in addition to `MakeNSError`.

Reply:
Any chance, `error_msg` could be populated through? error code might be enough but error message comes handy to provide debug info.

Reply:
If we've been doing 'make return type', then I prefer that for consistencies sake (and have done that here.)

(Though I like 'to return type' slightly better.)

Reply:
error_msg should already be populated by the WrapNSError/MakeNSError call. (StatusOr<T> contains both a code and an error message.)